ghost/core/test/unit/server/services/stats/posts.test.js

```
@@ -979,15 +979,15 @@ describe('PostsStatsService', function () {
-            assert.ok(Array.isArray(result), 'Result should be an array');
+            assert.ok(result.data && Array.isArray(result.data), 'Result should have data array');
             
             // With current implementation and date filtering issues, we expect posts but with 0 members
             // This test mainly verifies the method structure works correctly
-            if (result.length > 0) {
-                assert.ok(result[0].hasOwnProperty('post_id'), 'Results should have post_id');
-                assert.ok(result[0].hasOwnProperty('title'), 'Results should have title');
-                assert.ok(result[0].hasOwnProperty('views'), 'Results should have views');
-                assert.ok(result[0].hasOwnProperty('members'), 'Results should have members');
+            if (result.data.length > 0) {
+                assert.ok(result.data[0].hasOwnProperty('post_id'), 'Results should have post_id');
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)